docs/openapi.yaml

```diff
@@ -7,14 +7,67 @@ servers:
-
-    It is comprised of two parts; the Stacks Blockchain API and the Stacks Core API.
-
-    <a href="collection.json" download="collection.json">Download Postman collection</a>
+    Welcome to the API reference overview for the <a href="https://docs.hiro.so/get-started/stacks-blockchain-api">Stacks Blockchain API</a>.
+
+    <a href="https://hirosystems.github.io/stacks-blockchain-api/collection.json" download="stacks-api-collection.json">Download Postman collection</a>
+
+tags:
+  - name: Accounts
+    description: Read-only endpoints to obtain Stacks account details
+    externalDocs:
+      description: Stacks Documentation - Accounts
+      url: https://docs.stacks.co/understand-stacks/accounts
+  - name: Blocks
+    description: Read-only endpoints to obtain Stacks block details
+  - name: Faucets
+    description: Endpoints to request STX or BTC tokens (not possible on Mainnet)
+  - name: Fees
+    description: Read-only endpoints to obtain fee details
+  - name: Info
+    description: Read-only endpoints to obtain network, Proof-of-Transfer, Stacking, STX token, and node information
+  - name: Microblocks
+    description: Read-only endpoints to obtain microblocks details
+    externalDocs:
+      description: Stacks Documentation - Microblocks
+      url: https://docs.stacks.co/understand-stacks/microblocks
+  - name: Names
+    description: Read-only endpoints realted to the Blockchain Naming System on Stacks
+    externalDocs:
+      description: Stacks Documentation - Blockchain Naming System
+      url: https://docs.stacks.co/build-apps/references/bns
+  - name: Proxies
+    description: Endpoints that are proxied to the Stacks Node RPC API. Expect a higher latency for these endpoints.
+    externalDocs:
+      description: Hiro Documentation - Proxied Stacks Node RPC API endpoints
+      url: https://docs.hiro.so/get-started/stacks-blockchain-api#proxied-stacks-node-rpc-api-endpoints
+  - name: Rosetta
+    description: Endpoints to support the Rosetta API open blockchain standard
+    externalDocs:
+      description: Hiro Documentation - Rosetta Support
+      url: https://docs.hiro.so/get-started/stacks-blockchain-api#rosetta-support
+  - name: Smart Contracts
+    description: Read-only endpoints to obtain Clarity smart contract details
+    externalDocs:
+      description: Stacks Documentation - Clarity Smart Contracts
+      url: https://docs.stacks.co/write-smart-contracts/overview
+  - name: Stacking Rewards
+    description: Read-only endpoints to obtain Stacking reward details
+    externalDocs:
+      description: Stacks Documentation - Stacking
+      url: https://docs.stacks.co/understand-stacks/stacking
+  - name: Transactions
+    description: Endpoints to obtain transaction details and to broadcast transactions to the network
+    externalDocs:
+      description: Hiro Documentation - Transactions
+      url: https://docs.hiro.so/get-started/transactions
+  - name: Tokens
```

Comment:
I think we should separate FTs from NFTs (or have additional specific tags instead of only 'Tokens'), given that we plan on adding a few new NFT endpoints and users are very specifically looking for those. Thoughts?

Reply:
I like this! will make the change